src/psd_tools/api/layers.py

```diff
@@ -65,6 +65,16 @@ def layer_id(self):
         """
         return self.tagged_blocks.get_data(Tag.LAYER_ID, -1)
 
+    def __invalidate_bbox(self):
```

Comment:
Should be `_invalidate_bbox`, not double underscores `__invalidate_bbox` as typically used in the python standard convention